src/main/java/de/tum/in/www1/artemis/repository/ExamRepository.java

```diff
@@ -49,17 +49,36 @@ public interface ExamRepository extends JpaRepository<Exam, Long> {
             """)
     List<Exam> findAllByEndDateGreaterThanEqual(@Param("date") ZonedDateTime date);
 
+    /**
+     * Query which fetches all the active exams for which the user is instructor.
+     *
+     * @param groups   user groups
+     * @param pageable Pageable
+     * @param fromDate date to start from
+     * @param toDate   date to end at
+     * @return Page with exams
+     */
+    @Query("""
+            SELECT e FROM Exam e
+            WHERE e.course.instructorGroupName IN :groups
+                AND e.exerciseGroups IS NOT EMPTY
+                AND e.visibleDate >= :fromDate
+                AND e.visibleDate <= :toDate
```

Comment:
```suggestion
            SELECT e 
            FROM Exam e
            WHERE e.course.instructorGroupName IN :groups
                AND e.exerciseGroups IS NOT EMPTY
                AND e.visibleDate >= :fromDate
                AND e.visibleDate <= :toDate
```
Additionally: Why do we filter exams that don't have any exercise groups? Isn't it confusing if an exam is missing even though the startDate is already over

Reply:
> Additionally: Why do we filter exams that don't have any exercise groups?

I removed exercise groups filtering as it makes more sense to also have those exams. 

>Isn't it confusing if an exam is missing even though the startDate is already over

The exam won't be missing if startDate is already over.